rules-topics/14_transform-rituals.md

```diff
@@ -208,56 +221,64 @@ Alternatively, a character may still perform the Unlock Transform Abilities ritu
 |---|---|
 | Type: | Active |
 | Cost: | 10 + 10|
+| Tier: | 8 |
 | Description: | User may plan their feet, establish Concentration, and throw Harm Undead an unlimited number of times.  Effect ends after 5 minutes or after the user moves or disrupts their Concentration.|
 
 #### Body Weaponry
 | Name: | Body Weaponry|
 |---|---|
 | Type: | Passive |
 | Cost: | 20 (Single Purchase)|
-| Description: | User’s weapons are considered part of their body, and the user may, at their discretion, elect to take 20 points of damage in lieu of letting a weapon suffer from a destruction effect. This may designate all weapons, specific weapons, or shields based on the transform.|
+| Tier: | 8 |
+| Description: | User's weapons are considered part of their body, and the user may, at their discretion, elect to take 20 points of damage in lieu of letting a weapon suffer from a destruction effect. This may designate all weapons, specific weapons, or shields based on the transform.|
 
 #### Destroy Ritual Magic
 | Name: | Destroy Ritual Magic|
 |---|---|
 | Type: | Active |
 | Cost: | 20 (Single Purchase)|
+| Tier: | 8 |
 | Description: | Allows the user to destroy specific effects or products of ritual magic.  <i>Pending list.  Please see Plot to use this ability.</i>|
 
 #### Dodge
 | Name: | Dodge |
 |---|---|
 | Type: | Passive |
 | Cost: | 20 |
+| Tier: | 8 |
 | Description: | Reduces Defense Slot cost of Dodge by 1 (But cannot stack dodges) |
 
 #### Dull Pain
 | Name: | Dull Pain|
```

Comment:
Should this also now be Enhanced Focus or is the transform ability not changing?

Reply:
I am guessing we got caught up in updates because I did not change this.